.config/util.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+$FILE_ENV_VAR_SUFFIX = "_FILE";
```

Comment:
Could move this into function scope to avoid polluting the global scope, if that is a thing